CorsixTH/CMakeLists.txt

```diff
@@ -221,11 +223,12 @@ IF(APPLE)
     SET(BU_CHMOD_BUNDLE_ITEMS ON)
     FIXUP_BUNDLE(${CMAKE_INSTALL_PREFIX}/CorsixTH.app \"\" \"\")
     ")
-ELSE()
+else()
   install(TARGETS CorsixTH RUNTIME DESTINATION CorsixTH)
   install(DIRECTORY Campaigns Lua Levels DESTINATION CorsixTH PATTERN "*.svn" EXCLUDE)
   install(DIRECTORY Bitmap DESTINATION CorsixTH
-        FILES_MATCHING REGEX ".*\\.(tab|pal|dat|png)$"
-        PATTERN "*.svn" EXCLUDE)
+    FILES_MATCHING REGEX ".*\\.(tab|pal|dat|png)$"
+    PATTERN "*.svn" EXCLUDE
```

Comment:
Do we need to explicitly exclude SVN files or is this a holdover from before the project used git?

Reply:
Hold over, I can drop these.